src/main/java/io/jenkins/plugins/neuvector/NeuVectorBuilder.java

```diff
@@ -240,7 +243,7 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
         NeuVectorWorker worker = new NeuVectorWorker(logger, config, workspace, artifactName);
         try{
             worker.scan(run, launcher);
-        }catch (VulnerabilityCriterionException ex) {
+        }catch (VulnerabilityCriterionException | NoSuchAlgorithmException | KeyStoreException | KeyManagementException ex) {
```

Comment:
I think if you are not handling these Exceptions differently, you can just catch Exception so that you won't miss any Exceptions.